backends/dpdk/CMakeLists.txt

```diff
@@ -28,7 +28,7 @@ add_custom_target (dpdk_runtime_dir
 # placed in the build directory inside `control-plane` directory
 add_custom_command(
     OUTPUT ${DPDK_P4RUNTIME_INFO_GEN_SRCS} ${DPDK_P4RUNTIME_INFO_GEN_HDRS}
-  COMMAND ${PROTOBUF_PROTOC_EXECUTABLE}
+  COMMAND ${Protobuf_PROTOC_EXECUTABLE}
```

Comment:
The canonical Protobuf variable prefix is now `Protobuf_` instead of `PROTOBUF_`.

https://cmake.org/cmake/help/latest/module/FindProtobuf.html